xml/x11.xml

```diff
@@ -179,6 +179,226 @@ enabled=true</screen>
+        activated automatically.
+        </para>
+        </step>
+        <step>
+        <para>
+        Go to your &gnome; desktop. The gTile icon
+        is now shown in the right corner of the top bar.
+        </para>
+        </step>
+        </procedure>
+        <procedure>
+        <title>Installing from the command line</title>
+        <step>
+        <para>
+        Go to
+        <link xlink:href="https://extensions.gnome.org/extension/28/gtile/"></link>
+        and select the your <guimenu>Shell version</guimenu> and the
+        <guimenu>Extension version</guimenu> from the drop-down lists.
+        </para>
+        <para>
+        The extension is downloaded as <filename>.zip</filename> file.
+        </para>
+        </step>
+        <step>
+        <para>
+        In your command line, enter the following command
+        <command>gnome-extensions install <replaceable>FILENAME</replaceable>
+        </command>.
+        </para>
+        </step>
+        <step>
+        <para>
+        Go to your &gnome; desktop. The gTile icon
+        is now shown in the right corner of the top bar.
+        </para>
+        </step>
+        </procedure>
+        <note>
+        <title>System installation</title>
+        <para>
+        For a system installation, refer to
+        <link xlink:href="https://help.gnome.org/admin/system-admin-guide/stable/software.html.en#extension"></link>.
+        </para>
+        </note>
+        -->
+    </sect3>
+    <sect3>
+      <title>Working with the gTile extension</title>
+      <procedure>
+        <title>Arranging the windows</title>
+        <step>
+          <para>
+            To open gTile for the currently focused window, press
+            <keycombo><keycap function="meta"></keycap><keycap function="enter"></keycap></keycombo>.
+          </para>
+        </step>
+        <step>
+          <para>
+            To set the number of columns in which the windows will be arranged,
+            press a number key.
+          </para>
+        </step>
+        <step>
+          <para>
+            To automatically tile the windows according to the <guimenu>Autotile
+            Main window sizes</guimenu> setting, press the <keycap>M</keycap>
+            key. By default, the first time you press the <keycap>M</keycap> key,
+            it will make the current window use half the screen, and all other
+            windows to be tiled in a column on the other half of the screen. You
+            can press <keycap>M</keycap> repeatedly, to cycle between variants.
+            This is helpful to keep a big main window, and a number of smaller
+            secondary windows.
+          </para>
+        </step>
+        <step>
+          <para>
+            To move the current window to within the pre-defined tiling of the
+            screen, use the arrow keys.
+          </para>
+          <para>
+            To shrink or grow the number of tiles, use
```

Comment:
"the number of tiles that the window occupies", perhaps?  To clarify that it doesn't change the number of available tiles, just the window size.